apps/web-mzima-client/src/app/post/post-head/post-head.component.html

```diff
@@ -25,9 +27,10 @@
     *ngIf="!!user?.userId"
     fill="clear"
     color="gray"
-    tabindex="-1"
+    ariaLabel="More options"
```

Comment:
Is there a reason why you removed the tabindex here?

Reply:
tabindex="-1" here prevented the more options icon-button from receiving focus when navigating using only keyboard. Having it here will prevent keyboard users from ever reaching the more options menu for each post cards